charts/prefect-server/values.yaml

```
@@ -65,14 +59,11 @@ server:
   # sets PREFECT_LOGGING_SERVER_LEVEL
   loggingLevel: WARNING
 
-  ## If you want to run the UI seperately from the server, you will need to set the UI-specific URLs below
   uiConfig:
     # -- set PREFECT_UI_ENABLED; enable the UI on the server
     enabled: true
-    # -- sets PREFECT_UI_API_URL
-    prefectUiApiUrl: ""
-    # -- sets PREFECT_UI_URL
-    prefectUiUrl: ""
+    # -- sets PREFECT_UI_API_URL; If you want to connect to the UI from somewhere external to the cluster (i.e. via an ingress), you need to set this value to the ingress URL (e.g. http://app.internal.prefect.com/api).
```

```suggestion
    # -- sets PREFECT_UI_API_URL; If you want to connect to the UI from somewhere external to the cluster (i.e. via an ingress), you need to set this value to the ingress URL (e.g. http://app.internal.prefect.com/api).
```

- What do you think about linking to https://docs.prefect.io/v3/develop/settings-ref#api-url-2 for reference?
- That page says `The connection url for communication from the UI to the API.` This makes it sound like we could _also_ use this setting if the UI were running next to the API / in the same cluster, not necessarily from outside the cluster. Is that accurate?

---

I found those settings docs confusing, because it doesn't appear that `PREFECT_UI_API_URL` is the value we _should_ be using. @desertaxle what are your thoughts here?

---

Maybe link to these docs? https://docs.prefect.io/v3/manage/self-host#ui
That could still use some improvement, but it'll be easier to navigate than the settings reference.